common/annotations/sr_Latn.xml

```diff
@@ -1426,7 +1426,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🧑‍🍼" type="tts">osoba hrani bebu</annotation>
 		<annotation cp="👼">anđeo | bajka | beba | lice | mašta</annotation>
 		<annotation cp="👼" type="tts">beba anđeo</annotation>
-		<annotation cp="🎅">bajka | božić | božić bata | Deda Mraz | proslava</annotation>
+		<annotation cp="🎅">bajka | božić | božić bata | deda mraz | Deda Mraz | proslava</annotation>
```

Comment:
I think you also want to run -fP, which should remove the case duplicates here.

Reply:
`-fP` did not remove the case duplicates here.

Reply:
Hmmm. I'll file a ticket for that.

Reply:
FYI, a later CLDRModify pass did remove the case dup here